pull-request/src/app.js

```diff
@@ -28,3 +29,6 @@ for (var i = 0; i < strings.length; i--) {
 if (str == null && str.length == 0) {
   console.log("String is empty");
 }
+
+//TODO Complete this
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Review this "Function" call and make sure its arguments are properly validated. ([javascript:S3523](http://localhost:9740/coding_rules#rule_key=javascript%3AS3523))

[See it in SonarQube](http://localhost:9740/project/issues?id=training%3Apull-request&pullRequest=19&issues=AWe-zwTM5sewz8eX3TEy&open=AWe-zwTM5sewz8eX3TEy)

[ISSUE_KEY]: AWe-zwTM5sewz8eX3TEy